### PR TITLE
remove analysis tab from authenticated file

### PR DESCRIPTION
**Pull request by tcnichol**
Removing the Analysis tab from authenticated files. The user should not have access to that tab, matching behavior with datasts.

### CODE REVIEW

File: frontend/src/components/files/File.tsx [hunk 1]
```diff
@@ -115,7 +118,7 @@ export const File = (): JSX.Element => {
 	// snack bar
 	const [snackBarOpen, setSnackBarOpen] = useState(false);
 	const [snackBarMessage, setSnackBarMessage] = useState("");
-
+	console.log('file role', fileRole);
```
Comment (longshuicy) [inline frontend/src/components/files/File.tsx:121]: could you remove the console.log?

File: frontend/src/components/files/File.tsx [hunk 2]
```diff
@@ -343,14 +346,18 @@ export const File = (): JSX.Element => {
...
 						/>
-						<Tab
-							icon={<BuildIcon />}
-							iconPosition="start"
-							sx={TabStyle}
-							label="Analysis"
-							{...a11yProps(3)}
-							disabled={false}
-						/>
+						{datasetRole.role !== undefined && datasetRole.role !== "viewer" ? (
```
Comment (longshuicy) [inline frontend/src/components/files/File.tsx:349]: Could we use the AuthWrapper here?

File: frontend/src/components/files/File.tsx [hunk 3]
```diff
@@ -435,9 +442,13 @@ export const File = (): JSX.Element => {
 							version={fileSummary.version_num}
 						/>
 					</TabPanel>
-					<TabPanel value={selectedTabIndex} index={3}>
-						<Listeners fileId={fileId} datasetId={datasetId} />
-					</TabPanel>
+					{datasetRole.role !== undefined && datasetRole.role !== "viewer" ? (
```
Comment (longshuicy) [inline frontend/src/components/files/File.tsx:445]: Same could we use the "AuthWrapper" component here to wrap around?

**Review by longshuicy — CHANGES_REQUESTED** [review]

File: frontend/src/components/files/File.tsx (hunk 1 above)
Comment (tcnichol) [reply frontend/src/components/files/File.tsx]: removed

File: frontend/src/components/files/File.tsx (hunk 2 above)
Comment (tcnichol) [reply frontend/src/components/files/File.tsx]: changed to authwrapper

File: frontend/src/components/files/File.tsx (hunk 3 above)
Comment (tcnichol) [reply frontend/src/components/files/File.tsx]: changed here too

**Review by longshuicy — APPROVED** [review]
Works.

**Comment by tcnichol** [comment]
Merged beta into this branch.

**Pull request merged**